src/api-engine/api/lib/peer/channel.py

```diff
@@ -37,12 +37,38 @@ def list(self):
             raise Exception(err_msg)
         return res
 
-    def update(self, channel, channel_tx, orderer_url, time_out="90s"):
+    def update(self, channel, channel_tx, orderer_url):
+        """
+        Send a configtx update.
+
+        params:
+            channel: channel id.
+            channel_tx: Configuration transaction file generated by a tool such as configtxgen for submitting to orderer
+            orderer_url: Ordering service endpoint.
+        """
         try:
-            res = os.system("{} channel create -c {}  -f {} -o {} --timeout {}"
-                            .format(self.peer, channel, channel_tx, orderer_url, time_out))
+            res = os.system("{} channel update -c {}  -f {} -o {}"
```

Comment:
There is no such `channel update` cmd.

Reply:
Sorry, I am a little bit confused about it, I implemented it by the following [instructions](https://hyperledger-fabric.readthedocs.io/en/release-2.2/commands/peerchannel.html#peer-channel-update), it is `peer channel update` cmd and `self.peer` attribute indicates the path to peer CLI.

Reply:
oh, my bad. 
got your code now.